### DISCUSSION TITLE
Categories shown in dropdown of Upload Content

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Knowlg/Community#85] [General Discussions]
Hello All,

I came across the Upload content feature on the Sunbird ED where I can see a dropdown with a few categories

![preprodupont1](https://user-images.githubusercontent.com/65657373/211798791-6015c381-af8f-4af0-8b70-f84ce72d662e.png)

I want to know the if supported content is only
> File accepted - pdf, mp4, webm, epub, youtube, h5p and html zip

Then what is the use of  `Practice Question Set` and `Exam Question`.

Can anyone please help me understand this ?

**Comment by kameshbhr** [comment]
The list shown in Content Types comes from the primary categories configured in the system.  A primary category represents what the content is from a domain standpoint. This is entirely specific to an adopter and is configurable.  One can have the list of practice questions in the form of a pdf and label it as a "practice question set". All the uploaded files use underlying generalized asset type called "content".
A practice question set can also be a set of interactive questions, in which case it has to be created using underlying generalized asset called "question set". 
You can get more clarity of difference between generalized asset types and the domain specific categories here: https://knowlg.sunbird.org/learn/capabilities/rich-and-diverse-assets

**Reply by Xitija** [reply]
So the Practise Question Set need not be always a generalized asset that can be made by configuration on Sunbird , but can also take a form of a PDF or any other file . My understanding to this was that the Practise Question Set Primary category is only limited to the configurable generalized type. 
Thank you so much for the support.

**Reply by Krishnaj20** [reply]
@Xitija - you are confusing 'practice question set' which is a primary category belong to asset type 'content' and the actual content format which can be 'pdf' 'video' etc. Hope this answers your query.

**Reply by Xitija** [reply]
@Krishnaj20 
Sir for me Question Sets mean the content is of type QUML , and that is enabled through a Primary Category.
If I am configuring a Question Set of QUML type to the Primary Category of Practise Question Set then how can I upload a video under the same is something I am confused with.
Sorry If I am missing something obvious but I am trying to understand whats going on behind the scenes.
cc : @coolbung

**Reply by parthlawate** [reply]
I think the naming of the Content Category might be confusing here.. The Content 'Type' here is more adopter specific categories.. In this case .. these seem to be Diksha specific..  These could be anything you want.  So i could add things like 

When we say an actual Question Item/ Question Set - QUML -- thats the format of content..

**Reply by Krishnaj20** [reply]
thanks @parthlawate @kameshbhr .

@Xitija - I'm locking the conversation as your query is now answered.
